applications/template/cookiecutter.json

```diff
@@ -0,0 +1,14 @@
+{
+    "project_name": "My Project",
+    "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
+    "full_name": "John Doe",
+    "affiliation": "My Institution",
+    "language": "{{ cookiecutter.language }}",
+    "version": "0.1.0",
+    "holoscan_version": "2.3",
```

Comment:
I wonder if we could parse and default to the latest Holoscan SDK release?

Reply:
There are a few places we can pull this information from, open to opinions on the best source:
- Agreed `holoscan.__version__` is simplest, though it requires the user has the latest `holoscan` package installed locally. Maybe there is a good way such as `pip search` to find the latest version on PyPI without installing it?
- We currently inline this information in HoloHub CLI [here](https://github.com/nvidia-holoscan/holohub/blob/main/utilities/cli/container.py#L48), could expose the expected container version as a string staticmethod that we continue to manually update
- If a `holoscan` C++ install tree is available in the local environment (apt/container) we could parse `holoscan-config-version.cmake`

Reply:
thanks, for now I go for the 2nd option to add a unified variable to hold this info.